int32 seed
---
nav_msgs/OccupancyGrid map
